templates/engage/index.html

```diff
@@ -1072,5 +1072,14 @@ <h1>Ubuntu case studies, whitepapers and webinars</h1>
       {% endwith %}
     </div>
 
+    <div class="row u-equal-height u-clearfix">
```

Comment:
Asking because I don't know - the page itself is set to `noindex`, so will crawlers ultimately ignore this link?

Reply:
Ah good point I need to add nofollow to it.